### ISSUE TITLE
Unable to reproduce results for the Elasticity dataset

### ISSUE THREAD

**Issue by nnnfffkkk** [M3RG-IITD/LaMO#2]
Hello there

I am trying to reproduce the results reported in the paper but after multiple runs I don't think I am able to achieve what's reported in the paper. I am getting around 0.007 for the point cloud dataset.

May I confirm if you used a batch size of 1 without gradient accumulation?

**Comment by Karn3003** [comment]
Hello,

Thanks for reaching out. Yes, we used a batch size of 1 without gradient accumulation, as specified in the script/config. The discrepancy you’re seeing might be due to differences in the software environment or hardware (e.g., GPU card and CUDA version). Our reported results are based on the best mean over multiple runs. The performance varies across runs due to stochasticity; for example, in independent runs (3 runs), we obtained values of 0.00522, 0.00606, and 0.00509 on the point cloud dataset with train loss values of 0.00273, 0.0033, and 0.00265, respectively. Additionally, for Transolver, the elasticity value ranges from 0.0068 to 0.0092 across multiple runs.
